AdyenActions/AdyenActionComponent.swift

```diff
@@ -14,31 +14,48 @@ import UIKit
  - SeeAlso:
  [Implementation Reference](https://github.com/Adyen/adyen-ios#handling-an-action)
  */
-public final class AdyenActionComponent: ActionComponent, Localizable {
+public final class AdyenActionComponent: ActionComponent {
     
-    /// :nodoc:
     public let apiContext: APIContext
     
-    /// :nodoc:
     public weak var delegate: ActionComponentDelegate?
     
-    /// :nodoc:
     public weak var presentationDelegate: PresentationDelegate?
     
-    /// :nodoc:
-    public let style: ActionComponentStyle
-
-    /// :nodoc:
-    public var localizationParameters: LocalizationParameters?
+    /// Action handling configurations.
+    public var configuration: Configuration
     
+    /// Action handling configurations.
+    public struct Configuration: Localizable {
+        
+        /// Localization parameters.
+        public var localizationParameters: LocalizationParameters?
+        
+        /// The UI style configurations.
+        public var style: ActionComponentStyle = .init()
+        
+        /// Three DS configurations
+        public var threeDS: ThreeDS = .init()
+        
+        /// Three DS configurations
+        public struct ThreeDS {
+            /// `threeDSRequestorAppURL` for protocol version 2.2.0 OOB challenges
+            public var requestorAppURL: URL?
+            
+            public init() { /* Empty initializer */ }
```

Comment:
I think there's no need for an empty init. You get one by default.

Anyway, if you prefer to keep it, maybe adding // :nodoc: or some docs

Reply:
the default init is internal, for Public we have to provide it :-)

I will add `// :nodoc:`

Reply:
sorry, I added a comment `/// Initializes a new instance`, makes more sense I think